templates/databricks-multi-workspace-oem.template.yaml

```diff
@@ -810,7 +849,15 @@ Resources:
             Action: sts:AssumeRole
       ManagedPolicyArns:
         - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole 
-        - arn:aws:iam::aws:policy/IAMFullAccess 
+        - arn:aws:iam::aws:policy/IAMFullAccess
```

Comment:
why do we need IAMFullAccess?

Reply:
I think I can also reduce the scope to iam::put-role-policy only required by the lambda function

Reply:
Yes please. Please update to the policy with least privelege.